src/SwedbankPay.Sdk/SwedbankPayClient.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace SwedbankPay.Sdk
+{
+    using SwedbankPay.Sdk.Consumers;
+    using SwedbankPay.Sdk.PaymentOrders;
+    using SwedbankPay.Sdk.Payments;
+
+    using System.Net;
+
+    public class SwedbankPayClient : ISwedbankPayClient
+    {
+        public IPaymentOrdersResource PaymentOrders { get; }
+        public IPaymentsResource Payment { get; }
+        public IConsumersResource Consumers { get; }
+
+        public SwedbankPayClient(SwedbankPayOptions swedbankPayOptions, ILogSwedbankPayHttpResponse logger = null)
+        {
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
Is it possible to set `SecurityProtocol` to `Tls12` only if it is set to something lower than `Tls12` and not otherwise?